BossMod/Data/ClientState.cs

```diff
@@ -50,16 +50,28 @@ public record struct Stats(int SkillSpeed, int SpellSpeed, int Haste);
     public Stats PlayerStats;
     public readonly Cooldown[] Cooldowns = new Cooldown[NumCooldownGroups];
     public readonly ActionID[] DutyActions = new ActionID[2];
+    public readonly byte[] DutyActionCharges = new byte[2];
```

Comment:
I think this should actually be a single structure (with 2 action id's and 2 charge counts) rather than two different two-element arrays...

Reply:
agreed, i didn't know we had versioned ops. i can change that